concourse/pipelines/create-deployer.yml

```diff
@@ -208,7 +208,7 @@ jobs:
 
     - task: vpc-terraform-outputs-to-sh
       config:
-        image: docker:///ruby#2.2.3-slim
+        image: docker:///ruby#2.2-slim
```

Comment:
See the commit message for full details, but ruby uses a semver style versioning scheme.